authlogic.gemspec

```diff
@@ -18,16 +18,16 @@ require "authlogic/version"
     "tieg.zaharia@gmail.com",
     "jared@jaredbeck.com"
   ]
-  s.homepage = "http://github.com/binarylogic/authlogic"
+  s.homepage = "https://github.com/binarylogic/authlogic"
   s.summary = "An unobtrusive ruby authentication library based on ActiveRecord."
   s.license = "MIT"
 
-  s.required_ruby_version = ">= 2.4.0"
+  s.required_ruby_version = ">= 2.7.0"
 
   # See doc/rails_support_in_authlogic_5.0.md
-  s.add_dependency "activemodel", [">= 5.2", "< 6.2"]
-  s.add_dependency "activerecord", [">= 5.2", "< 6.2"]
-  s.add_dependency "activesupport", [">= 5.2", "< 6.2"]
+  s.add_dependency "activemodel", [">= 5.2", "<= 7.0"]
+  s.add_dependency "activerecord", [">= 5.2", "<= 7.0"]
+  s.add_dependency "activesupport", [">= 5.2", "<= 7.0"]
```

Comment:
Should we specify `"< 7.1"`, so that e.g. `7.0.1` is allowed?

Reply:
I agree to all the changes.